src/main/java/org/example/service/UserService.java

```diff
@@ -0,0 +1,6 @@
+package org.example.service;
+
+public interface UserService {
+    UserResponseDto registerUser(UserRegistrationDto userRegistrationDto);
+    UserResponseDto getUserById(Long userId);
```

Comment:
You will call this method from the UserService instance. You already know that you are working with users, not cars, payments, etc..., so method name may looks like:
```suggestion
    UserResponseDto getById(Long userId);
```